en/enable-tls-between-components.md

```diff
@@ -15,6 +15,10 @@ To enable TLS between TiDB components, perform the following steps:
    - A set of server-side certificates for the PD/TiKV/TiDB/Pump/Drainer/TiFlash/TiKV Importer/TiDB Lightning component, saved as the Kubernetes Secret objects: `${cluster_name}-${component_name}-cluster-secret`.
    - A set of shared client-side certificates for the various clients of each component, saved as the Kubernetes Secret objects: `${cluster_name}-cluster-client-secret`.
 
+    > **Note:**
+    >
+    > The Secret objects you created must follow the above naming convention. Otherwise, the deployment of the Components will fail.
```

Comment:
```suggestion
    > The Secret objects you created must follow the above naming convention. Otherwise, the deployment of the TiDB components will fail.
```